lib/pdf.js

```diff
@@ -35,6 +35,12 @@ function PDF (html, options) {
   if (this.options.filename) this.options.filename = path.resolve(this.options.filename)
   if (!this.options.phantomPath) this.options.phantomPath = phantomjs && phantomjs.path
   this.options.phantomArgs = this.options.phantomArgs || []
+  this.options.phantomArgs = this.options.phantomArgs.map(function (elt) {
+    if (elt.substring(0, 2) !== '--') {
```

Comment:
I agree with you, it would be better but unfortunately it does not work like this.
The double quotes are not kept and yargs-parser consider it as a normal argument.